less/_defaults/_font-config.less

```diff
@@ -121,7 +121,7 @@
 
 @page-instruction-size: @page-body-size;
 @page-instruction-family: @body-family;
```

Comment:
Can you update to `@page-instruction-family: @page-body-family;` to keep inline with `@menu-instruction-family` please

Reply:
Good spot